src/services/global.js

```diff
@@ -1,10 +1,8 @@
 import request from '../utils/request';
 
-const { endpoints } = window;
-
 export async function saveUserSession(params) {
   const { sessionConfig, description } = params;
-  const endpoint = `${endpoints.api.graphql}`;
+  const endpoint = `${window.endpoints.api.graphql}`;
```

Comment:
You don't need the backtick string substitution here when `const endpoint = window.endpoints.api.graphql` would work just as well. (And in fact, the `endpoint` local really doesn't help much anymore; it's already been removed in most other paths.)